### PR TITLE
[fix bug 1271422] Clean up old /contribute/signup/ code & assets

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Removes old unused /contribute templates and associated assets & tests.
- Removes commented out code from contribute pages that was never used.
- Slight refactoring/renaming of base JS and CSS templates to remove unused code.
- Changes `/contribute/embed/` form to a simple heading & link back to mozorg.

## Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1271422

## Testing
- I ran integration tests under `/functional/contribute/` and all pass OK (I'm going to add tests for the new signup & task pages separately to this PR).
- As this change removes & renames certain assets, it may be useful to test this branch with a clean `/static/` directory.

## Checklist
- [x] Requires l10n changes.
- [x] Related functional & integration tests passing.

### CODE REVIEW

**Comment by flodolo** [comment]
Marking as L10N because we'll need to clean up files on our side too.

I'm pretty sure you can rid of good part of this file (or maybe entirely)
https://github.com/mozilla/bedrock/blob/master/bedrock/mozorg/email_contribute.py

**Comment by alexgibson** [comment]
> I'm pretty sure you can rid of good part of this file (or maybe entirely)
https://github.com/mozilla/bedrock/blob/master/bedrock/mozorg/email_contribute.py

Good spot, thanks @flodolo 👍

**Comment by alexgibson** [comment]
Ok I think this one is ready for review. Going to keep the do-not-merge flag for now until I double check the changed for `contribute/embed` are OK for now.

**Comment by alexgibson** [comment]
Got the [OK in the bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1271422#c9) re: changes to `/contribute/embed/`. Removing the do-not-merge label.

**Comment by alexgibson** [comment]
Adding back the do not merge flag for now, as there is some discussion the bugs out the embed form still being used by the community.

**Comment by alexgibson** [comment]
@jpetto - from discussion in the bug I'm going to say that this is safe enough to proceed with code review. It sounds like the embed form gets very little traffic. I'm going to leave the do-not-merge label for now, just until we get the final green light.

**Comment by alexgibson** [comment]
Latest in the bug is that this PR will remain blocked until July 1, at which point we can proceed with removing the old form(s).

File: bedrock/mozorg/urls.py [hunk 1]
```diff
@@ -197,39 +197,17 @@
...
-        kwargs={'template': 'mozorg/contribute/index.html',
-                'return_to_form': True},
-        name='mozorg.contribute_event'),
-    url('^contribute/embed/$', views.contribute_embed,
-        name='mozorg.contribute_embed',
-        kwargs={'template': 'mozorg/contribute/contribute-embed.html',
-                'return_to_form': False}),
-
-    page('contribute/signup', 'mozorg/contribute/signup.html'),
-    url('^contribute/task/',
```
Comment (jpetto) [inline bedrock/mozorg/urls.py]: I'm getting an error locally navigating to `/contribute/signup/`. I think we still need this URL resolved for the individual tasks.

**Comment by jpetto** [comment]
Aside from the (probably accidentally deleted) missing URL resolver, this is looking good. Tested with a clean `/static` directory in modern browsers and IE 8. As far as I can tell, all refactoring and deletions are solid.

r+wc

File: bedrock/mozorg/urls.py (hunk 1 above)
Comment (alexgibson) [inline bedrock/mozorg/urls.py]: Gah, the perils of rebasing a large branch over a long period. Will fix!

**Comment by alexgibson** [comment]
@jpetto - thanks for the review, updated.

**Comment by jpetto** [comment]
🍔 🍟 👍

**Pull request merged**